src/GapsInCareHelpers.ts

```diff
@@ -371,20 +293,20 @@ export function generateGapsInCareBundle(
  * @param detailedResult result from calculation to look up clause values
  * @return mapped list of queries with near miss info if relevant
```

Comment:
```suggestion
 * @return mapped list of queries with reason detail info if relevant
```

Reply:
Also, above: "Add near miss data to each DataTypeQuery passed in" -> "Add reason detail data to each DataTypeQuery passed in" (line 289, won't let me directly suggest it because it's too far from the changed line)